lessons/misc/webdev-intro/app/slides-module.js

```diff
@@ -0,0 +1,270 @@
+'use strict';
```

Comment:
I don't think the meaning of this (supposedly useless) line is mentioned anywhere.